src/Helper/Order.php

```diff
@@ -141,6 +140,18 @@ public function loadMethods(array $methods)
         return $this->mageOrder;
     }
 
+    /**
+     * Executes after order is fully loaded and updated
+     */
+    public function setEndUpdate()
+    {
+        $this->mageOrder->addStatusHistoryComment(__('[SHOPGATE] Order updated by Shopgate.'))
```

Comment:
have u by chance tested this? Especially translation.

Reply:
Not the update. Code was just moved, that's why it appears in the change log.

Reply:
ya, i know, wut about the create. It should have one too, right?

Reply:
You mean 'Order added by Shopgate'? If so, it's already done in setOrderStatusHistory()